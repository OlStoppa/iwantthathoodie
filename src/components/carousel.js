import React, {useEffect, useState } from "react"
import { StaticQuery, graphql } from "gatsby"
import Img from "gatsby-image"

const Carousel = ({ data }) => {
    
    const [displayedImage, incrementDisplayed] = useState(0)
    
    
    

    useEffect(() => {
        const started = setInterval(() => {
            
            incrementDisplayed( displayedImage => displayedImage < data.CarouselImages.edges.length -1 ? displayedImage + 1 : 0)
            
        }, 8000)
        return () => { clearInterval(started) }
    }, [] )
      
       
    
        let next = displayedImage < data.CarouselImages.edges.length - 1 ? displayedImage + 1 : 0
    
    return (
        <>
        {data.CarouselImages.edges.map((image, index) => {
            let style;
            if(index === displayedImage){
                 style = "top"
            }
            else if(index === next){
                 style = "next"
            }else {
                 style = ""
            }

        return <div className={`top-image-container ${style}`}>
        <Img imgStyle={{opacity: "inherit", transition: "inherit"}} fluid={data.CarouselImages.edges[index].node.childImageSharp.fluid}/>
        </div>})
        }
        
        
        </>
    )
}

export default props => (
    <StaticQuery
      query={graphql`
      query {
        CarouselImages: allFile(
          filter: {
            extension: { regex: "/(jpg)|(png)/" }
            relativeDirectory: { eq: "carousel" }
          }
        ) {
          edges {
            node {
              childImageSharp {
                resize(width: 1500, height: 1500) {
                  src
                }
                fluid(maxWidth: 1300) {
                  ...GatsbyImageSharpFluid
                }
                
              }
            }
          }
        }
        
      }
      
  
      `}
      render={data => <Carousel data={data} {...props} />}
      />
  )