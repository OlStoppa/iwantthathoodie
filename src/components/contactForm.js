import React from "react"

const ContactForm = () => {
  return (
    <form name="contact" method="POST" data-netlify="true">
        <h2>Contact Us</h2>
      <p>
        <label>
          Your Name: <input placeholder="Name" type="text" name="name" />
        </label>
      </p>
      <p>
        <label>
          Your Email: <input placeholder="Email" type="email" name="email" />
        </label>
      </p>
      <p>
        <label>
          Message: <textarea placeholder="Your Message" name="message" />
        </label>
      </p>
      <p>
        <button type="submit">Send</button>
      </p>
    </form>
  )
}

export default ContactForm