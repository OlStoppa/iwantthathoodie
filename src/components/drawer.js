import React from "react"

import { Link } from "gatsby"


const Drawer = props => {
  

  return (
    <div className={props.isOpen ? "drawer open" : "drawer"}>
      
      <div className="mobile-nav-container">
        <ul>
         
          <Link to="/contact"><li>Contact Us</li></Link>
        </ul>
      </div>
    </div>
  )
}

export default Drawer
