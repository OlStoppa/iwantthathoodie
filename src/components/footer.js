import React from "react"


const Footer = () => (
    <div className="footer">
        <div className="footer--inner">
        Â© 2019 iwantthathoodie.com
        </div>
        
    </div>
)

export default Footer