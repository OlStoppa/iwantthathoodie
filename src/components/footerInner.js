import React from 'react'
import Image from '../components/image'
import Social from '../components/social'

const FooterInner = () => {
    return (
        <div className="footer-content">
            <Social/>
            <div className="footer-middle">
            <div className="footer-links">
                <ul>
                    <li>SHIPPING</li>
                    <li>RETURNS</li>
                    <li>CONTACT</li>
                    <li>ABOUT US</li>
                </ul>
            </div>
            <div className="paypal-image">
                <Image filename="paypal.png" />
            </div>
            
            </div>
            <div>
                Â© 2019 thishoodie.com
            </div>
            
            
        </div>
    )
}

export default FooterInner