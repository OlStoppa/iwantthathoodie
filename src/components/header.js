import { Link } from "gatsby"
import PropTypes from "prop-types"
import React, {useContext} from "react"
import  cartContext  from "../context/cartContext"
import Navbar from "./navbar"
import Logo from "../images/hoodie-logo.inline.svg"

const Header = ({ siteTitle }) => {

  const [state, dispatch] = useContext(cartContext) 
return(
  <header>
    <div
      style={{
        margin: `0 auto`,
        maxWidth: 1200,
        padding: `0.5rem 1.45rem 0rem`,
        display: 'flex',
        justifyContent: 'space-between'
      }}
    >
      
        <Link
          to="/"
          style={{
            color: `white`,
            textDecoration: `none`,
          }}
        >
          <Logo className="logo"/>
        </Link>
        
      
      <Link to="/cart">
      <div className="cart-icon__container">
        <span className="material-icons" >shopping_cart</span>
        {state.length > 0 && <span className="cart-counter">{state.reduce((a, b)  => a + b.qty,0 )}</span>}
      </div>
      </Link>
      
    </div>
    <Navbar />
  </header>
)
        }

Header.propTypes = {
  siteTitle: PropTypes.string,
}

Header.defaultProps = {
  siteTitle: ``,
}

export default Header
