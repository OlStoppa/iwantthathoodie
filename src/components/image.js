import React from "react"
import { StaticQuery, graphql } from "gatsby"
import Img from "gatsby-image"

export const PureImage = ({ data, filename, alt }) => {
  console.log(filename)

  const image = data.images.edges.find(n => {
    return n.node.relativePath.includes(filename)
  })
  if (!image) {
    return null
  }

  const imageSizes = image.node.childImageSharp.sizes
  return <Img alt={alt} sizes={imageSizes} />
}

export default props => (
  <StaticQuery
    query={graphql`
      query {
        images: allFile {
          edges {
            node {
              relativePath
              name
              childImageSharp {
                sizes(maxWidth: 1030) {
                  ...GatsbyImageSharpSizes
                }
              }
            }
          }
        }
      }
    `}
    render={data => <PureImage data={data} {...props} />}
  />
)
