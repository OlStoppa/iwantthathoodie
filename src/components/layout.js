/**
 * Layout component that queries for data
 * with Gatsby's StaticQuery component
 *
 * See: https://www.gatsbyjs.org/docs/static-query/
 */

import React, { useState } from "react"
import PropTypes from "prop-types"
import { StaticQuery, graphql } from "gatsby"
import Drawer from "./drawer"
import Header from "./header"
import Footer from "./footer"
import "./index.scss"

const Layout = ({ children }) => {
  const [drawerOpen, setDrawerOpen] = useState(false)

  const toggleDrawer = () => {
    setDrawerOpen(!drawerOpen)
  }
return(
  <StaticQuery
    query={graphql`
      query SiteTitleQuery {
        site {
          siteMetadata {
            title
          }
        }
      }
    `}

    
    render={data => (
      <div className="site-container">
        <Header 
        siteTitle={data.site.siteMetadata.title}
        toggleDrawer={toggleDrawer}
        isOpen={drawerOpen}
        />
        <Drawer
          isOpen={drawerOpen}
        />
        <div
          style={{
            margin: `0 auto`,
            maxWidth: 1200,
            padding: `0px 0px 1.45rem`,
            paddingTop: 0,
            flexGrow: 1
            
          }}
        >
          <main>{children}</main>
        </div>
        <Footer />
        {drawerOpen && 
        <div className="overlay" onClick={toggleDrawer}/>
        }
      </div>
    )}
  />
)
        }

Layout.propTypes = {
  children: PropTypes.node.isRequired,
}

export default Layout
