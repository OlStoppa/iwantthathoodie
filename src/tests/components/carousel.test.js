import React from 'react'
import {render} from '@testing-library/react'
import {PureCarousel as Carousel } from '../../components/carousel'

test('renders', () => {

    const data =  {
        "CarouselImages": {
          "edges": [
            {
              "node": {
                "childImageSharp": {
                  "resize": {
                    "src": "/static/3d918829ff7c56c4d1585f9db25d00db/d5c1a/daytime-facial-expression-fashion-859057.jpg"
                  },
                  "fluid": {
                    "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAGAAAAwEBAAAAAAAAAAAAAAAAAAIEAwX/xAAVAQEBAAAAAAAAAAAAAAAAAAAAAf/aAAwDAQACEAMQAAAB6uLQxSKH/8QAGhAAAwEAAwAAAAAAAAAAAAAAAAECAxMhIv/aAAgBAQABBQJFtql6JZrpXNPR/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAwEBPwE//8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAgEBPwE//8QAGhAAAgMBAQAAAAAAAAAAAAAAAAEREkEQgf/aAAgBAQAGPwITsq6ieRkpHp//xAAaEAEAAwADAAAAAAAAAAAAAAABABEhQVFh/9oACAEBAAE/ISVCIANCIOK4mMyNVoV3ME9T/9oADAMBAAIAAwAAABBs/wD/xAAXEQADAQAAAAAAAAAAAAAAAAABECEx/9oACAEDAQE/EBtX/8QAFREBAQAAAAAAAAAAAAAAAAAAAHH/2gAIAQIBAT8QV//EABsQAQACAwEBAAAAAAAAAAAAAAEAESExUUGB/9oACAEBAAE/EBDETAbYL43O7jS701FRiCTXErWBXPyJKzac2z//2Q==",
                    "aspectRatio": 1.5,
                    "src": "/static/3d918829ff7c56c4d1585f9db25d00db/91194/daytime-facial-expression-fashion-859057.jpg",
                    "srcSet": "/static/3d918829ff7c56c4d1585f9db25d00db/c6906/daytime-facial-expression-fashion-859057.jpg 325w,\n/static/3d918829ff7c56c4d1585f9db25d00db/00201/daytime-facial-expression-fashion-859057.jpg 650w,\n/static/3d918829ff7c56c4d1585f9db25d00db/91194/daytime-facial-expression-fashion-859057.jpg 1300w,\n/static/3d918829ff7c56c4d1585f9db25d00db/15715/daytime-facial-expression-fashion-859057.jpg 1950w,\n/static/3d918829ff7c56c4d1585f9db25d00db/f234e/daytime-facial-expression-fashion-859057.jpg 2600w,\n/static/3d918829ff7c56c4d1585f9db25d00db/962d5/daytime-facial-expression-fashion-859057.jpg 5265w",
                    "sizes": "(max-width: 1300px) 100vw, 1300px"
                  }
                }
              }
            },
            {
              "node": {
                "childImageSharp": {
                  "resize": {
                    "src": "/static/8565d81975436f1659bbbbf123d543ad/d5c1a/beautiful-bright-colours-colors-1158670.jpg"
                  },
                  "fluid": {
                    "base64": "data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAFwABAQEBAAAAAAAAAAAAAAAAAAUCBP/EABUBAQEAAAAAAAAAAAAAAAAAAAEA/9oADAMBAAIQAxAAAAGnxJo2GEf/xAAaEAEAAgMBAAAAAAAAAAAAAAABAgMRISMx/9oACAEBAAEFAjzLYzQbJcabklPb/8QAFREBAQAAAAAAAAAAAAAAAAAAARD/2gAIAQMBAT8BJ//EABQRAQAAAAAAAAAAAAAAAAAAABD/2gAIAQIBAT8BP//EABoQAAICAwAAAAAAAAAAAAAAAAABAhARMXH/2gAIAQEABj8CE1JrDqXDVf/EABkQAQADAQEAAAAAAAAAAAAAAAEAESExQf/aAAgBAQABPyGhZ4RBQBolbR5EnpCHhmewm8//2gAMAwEAAgADAAAAEPj/AP/EABYRAAMAAAAAAAAAAAAAAAAAABARQf/aAAgBAwEBPxBKP//EABURAQEAAAAAAAAAAAAAAAAAAAAh/9oACAECAQE/EFf/xAAcEAEAAwADAQEAAAAAAAAAAAABABEhUWFxofD/2gAIAQEAAT8QTkALXghmSFu2Gfu4ZQW2esy7Sov3IEKGaeEfsspbCse5/9k=",
                    "aspectRatio": 1.5,
                    "src": "/static/8565d81975436f1659bbbbf123d543ad/91194/beautiful-bright-colours-colors-1158670.jpg",
                    "srcSet": "/static/8565d81975436f1659bbbbf123d543ad/c6906/beautiful-bright-colours-colors-1158670.jpg 325w,\n/static/8565d81975436f1659bbbbf123d543ad/00201/beautiful-bright-colours-colors-1158670.jpg 650w,\n/static/8565d81975436f1659bbbbf123d543ad/91194/beautiful-bright-colours-colors-1158670.jpg 1300w,\n/static/8565d81975436f1659bbbbf123d543ad/15715/beautiful-bright-colours-colors-1158670.jpg 1950w,\n/static/8565d81975436f1659bbbbf123d543ad/f234e/beautiful-bright-colours-colors-1158670.jpg 2600w,\n/static/8565d81975436f1659bbbbf123d543ad/39b3c/beautiful-bright-colours-colors-1158670.jpg 5472w",
                    "sizes": "(max-width: 1300px) 100vw, 1300px"
                  }
                }
              }
            }
          ]
        }
      }
    const {asFragment} = render(<Carousel data={data}/>)
    expect(asFragment()).toMatchSnapshot();
} )