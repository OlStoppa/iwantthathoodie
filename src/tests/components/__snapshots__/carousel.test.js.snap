// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="top-image-container top"
  >
    <div
      class=" gatsby-image-wrapper"
      style="position: relative; overflow: hidden;"
    >
      <div
        style="width: 100%; padding-bottom: 66.66666666666667%;"
      />
      <img
        alt=""
        src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAGAAAAwEBAAAAAAAAAAAAAAAAAAIEAwX/xAAVAQEBAAAAAAAAAAAAAAAAAAAAAf/aAAwDAQACEAMQAAAB6uLQxSKH/8QAGhAAAwEAAwAAAAAAAAAAAAAAAAECAxMhIv/aAAgBAQABBQJFtql6JZrpXNPR/8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAwEBPwE//8QAFBEBAAAAAAAAAAAAAAAAAAAAEP/aAAgBAgEBPwE//8QAGhAAAgMBAQAAAAAAAAAAAAAAAAEREkEQgf/aAAgBAQAGPwITsq6ieRkpHp//xAAaEAEAAwADAAAAAAAAAAAAAAABABEhQVFh/9oACAEBAAE/ISVCIANCIOK4mMyNVoV3ME9T/9oADAMBAAIAAwAAABBs/wD/xAAXEQADAQAAAAAAAAAAAAAAAAABECEx/9oACAEDAQE/EBtX/8QAFREBAQAAAAAAAAAAAAAAAAAAAHH/2gAIAQIBAT8QV//EABsQAQACAwEBAAAAAAAAAAAAAAEAESExUUGB/9oACAEBAAE/EBDETAbYL43O7jS701FRiCTXErWBXPyJKzac2z//2Q=="
        style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; transition-delay: 500ms; transition: inherit;"
      />
      <picture>
        <img
          alt=""
          sizes="(max-width: 1300px) 100vw, 1300px"
          src="/static/3d918829ff7c56c4d1585f9db25d00db/91194/daytime-facial-expression-fashion-859057.jpg"
          srcset="/static/3d918829ff7c56c4d1585f9db25d00db/c6906/daytime-facial-expression-fashion-859057.jpg 325w,
/static/3d918829ff7c56c4d1585f9db25d00db/00201/daytime-facial-expression-fashion-859057.jpg 650w,
/static/3d918829ff7c56c4d1585f9db25d00db/91194/daytime-facial-expression-fashion-859057.jpg 1300w,
/static/3d918829ff7c56c4d1585f9db25d00db/15715/daytime-facial-expression-fashion-859057.jpg 1950w,
/static/3d918829ff7c56c4d1585f9db25d00db/f234e/daytime-facial-expression-fashion-859057.jpg 2600w,
/static/3d918829ff7c56c4d1585f9db25d00db/962d5/daytime-facial-expression-fashion-859057.jpg 5265w"
          style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; transition: inherit;"
        />
      </picture>
      <noscript>
        &lt;picture&gt;&lt;img loading="lazy" sizes="(max-width: 1300px) 100vw, 1300px" srcset="/static/3d918829ff7c56c4d1585f9db25d00db/c6906/daytime-facial-expression-fashion-859057.jpg 325w,
/static/3d918829ff7c56c4d1585f9db25d00db/00201/daytime-facial-expression-fashion-859057.jpg 650w,
/static/3d918829ff7c56c4d1585f9db25d00db/91194/daytime-facial-expression-fashion-859057.jpg 1300w,
/static/3d918829ff7c56c4d1585f9db25d00db/15715/daytime-facial-expression-fashion-859057.jpg 1950w,
/static/3d918829ff7c56c4d1585f9db25d00db/f234e/daytime-facial-expression-fashion-859057.jpg 2600w,
/static/3d918829ff7c56c4d1585f9db25d00db/962d5/daytime-facial-expression-fashion-859057.jpg 5265w" src="/static/3d918829ff7c56c4d1585f9db25d00db/91194/daytime-facial-expression-fashion-859057.jpg" alt="" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
      </noscript>
    </div>
  </div>
  <div
    class="top-image-container next"
  >
    <div
      class=" gatsby-image-wrapper"
      style="position: relative; overflow: hidden;"
    >
      <div
        style="width: 100%; padding-bottom: 66.66666666666667%;"
      />
      <img
        alt=""
        src="data:image/jpeg;base64,/9j/2wBDABALDA4MChAODQ4SERATGCgaGBYWGDEjJR0oOjM9PDkzODdASFxOQERXRTc4UG1RV19iZ2hnPk1xeXBkeFxlZ2P/2wBDARESEhgVGC8aGi9jQjhCY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2NjY2P/wgARCAANABQDASIAAhEBAxEB/8QAFwABAQEBAAAAAAAAAAAAAAAAAAUCBP/EABUBAQEAAAAAAAAAAAAAAAAAAAEA/9oADAMBAAIQAxAAAAGnxJo2GEf/xAAaEAEAAgMBAAAAAAAAAAAAAAABAgMRISMx/9oACAEBAAEFAjzLYzQbJcabklPb/8QAFREBAQAAAAAAAAAAAAAAAAAAARD/2gAIAQMBAT8BJ//EABQRAQAAAAAAAAAAAAAAAAAAABD/2gAIAQIBAT8BP//EABoQAAICAwAAAAAAAAAAAAAAAAABAhARMXH/2gAIAQEABj8CE1JrDqXDVf/EABkQAQADAQEAAAAAAAAAAAAAAAEAESExQf/aAAgBAQABPyGhZ4RBQBolbR5EnpCHhmewm8//2gAMAwEAAgADAAAAEPj/AP/EABYRAAMAAAAAAAAAAAAAAAAAABARQf/aAAgBAwEBPxBKP//EABURAQEAAAAAAAAAAAAAAAAAAAAh/9oACAECAQE/EFf/xAAcEAEAAwADAQEAAAAAAAAAAAABABEhUWFxofD/2gAIAQEAAT8QTkALXghmSFu2Gfu4ZQW2esy7Sov3IEKGaeEfsspbCse5/9k="
        style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; transition-delay: 500ms; transition: inherit;"
      />
      <picture>
        <img
          alt=""
          sizes="(max-width: 1300px) 100vw, 1300px"
          src="/static/8565d81975436f1659bbbbf123d543ad/91194/beautiful-bright-colours-colors-1158670.jpg"
          srcset="/static/8565d81975436f1659bbbbf123d543ad/c6906/beautiful-bright-colours-colors-1158670.jpg 325w,
/static/8565d81975436f1659bbbbf123d543ad/00201/beautiful-bright-colours-colors-1158670.jpg 650w,
/static/8565d81975436f1659bbbbf123d543ad/91194/beautiful-bright-colours-colors-1158670.jpg 1300w,
/static/8565d81975436f1659bbbbf123d543ad/15715/beautiful-bright-colours-colors-1158670.jpg 1950w,
/static/8565d81975436f1659bbbbf123d543ad/f234e/beautiful-bright-colours-colors-1158670.jpg 2600w,
/static/8565d81975436f1659bbbbf123d543ad/39b3c/beautiful-bright-colours-colors-1158670.jpg 5472w"
          style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; transition: inherit;"
        />
      </picture>
      <noscript>
        &lt;picture&gt;&lt;img loading="lazy" sizes="(max-width: 1300px) 100vw, 1300px" srcset="/static/8565d81975436f1659bbbbf123d543ad/c6906/beautiful-bright-colours-colors-1158670.jpg 325w,
/static/8565d81975436f1659bbbbf123d543ad/00201/beautiful-bright-colours-colors-1158670.jpg 650w,
/static/8565d81975436f1659bbbbf123d543ad/91194/beautiful-bright-colours-colors-1158670.jpg 1300w,
/static/8565d81975436f1659bbbbf123d543ad/15715/beautiful-bright-colours-colors-1158670.jpg 1950w,
/static/8565d81975436f1659bbbbf123d543ad/f234e/beautiful-bright-colours-colors-1158670.jpg 2600w,
/static/8565d81975436f1659bbbbf123d543ad/39b3c/beautiful-bright-colours-colors-1158670.jpg 5472w" src="/static/8565d81975436f1659bbbbf123d543ad/91194/beautiful-bright-colours-colors-1158670.jpg" alt="" style="position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center"/&gt;&lt;/picture&gt;
      </noscript>
    </div>
  </div>
</DocumentFragment>
`;
